Jenkinsfile

```diff
@@ -161,6 +188,7 @@ pipeline {
 
           gcloud container images add-tag gcr.io/kaggle-images/python:${PRETEST_TAG} gcr.io/kaggle-images/python:${STAGING_TAG}
           gcloud container images add-tag gcr.io/kaggle-private-byod/python:${PRETEST_TAG} gcr.io/kaggle-private-byod/python:${STAGING_TAG}
+          gcloud container images add-tag gcr.io/kaggle-private-byod/python-tpuvm:${PRETEST_TAG} gcr.io/kaggle-private-byod/python-tpuvm:${STAGING_TAG}
```

Comment:
The `staging` label should be added after the tests are passing. Should you run the tests here?

Reply:
I didn't have any tests setup yet, in part because we don't have a TPU attached in jenkins to test against, and otherwise it has basically the same libs as the CPU image (which gets tested already).

I would like to figure out a testing plan eventually, but figured I'd just start by matching the existing setup as much as possible.

Reply:
Makes sense. Thanks for the clarification.